pkg/networkmanager/interfaces.js

```diff
@@ -1556,11 +1556,9 @@ export function with_checkpoint(model, modify, options) {
                                         .catch(function () {
                                             show_breaking_change_dialog({
                                                 ...options,
-                                                action: () => {
-                                                    syn_click(model, function () {
-                                                        modify();
-                                                    });
-                                                }
+                                                action: syn_click(model, function () {
+                                                    modify();
+                                                }),
```

Comment:
Just leaves me curious -- why not go all the way and do `action: syn_click(model, modify)`?

Reply:
Ooo, good point done!